From ed081caa40c7dd904414daba14062f4e23a0ec8a Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Sat, 27 Jul 2013 17:05:18 +0000
Subject: [PATCH] add package.xml install rule and remove arch dependent stuff

---
 CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1dc20dc..b08ec09 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -190,10 +190,6 @@ IF(CMAKE_COMPILER_IS_GNUCXX)
     #SET(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE}") 
   #ENDIF(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   # Linux
-  IF(NOT ANDROID AND ${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-    SET(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -march=native") 
-    SET(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -march=native") 
-  ENDIF()
   # activate warnings !!!
   SET(g2o_C_FLAGS "${g2o_C_FLAGS} -Wall -W")
   SET(g2o_CXX_FLAGS "${g2o_CXX_FLAGS} -Wall -W")
@@ -240,3 +236,7 @@ INSTALL(FILES ${PROJECT_BINARY_DIR}/g2o/config.h DESTINATION ${CMAKE_INSTALL_PRE
 # Include the subdirectories
 ADD_SUBDIRECTORY(EXTERNAL)
 ADD_SUBDIRECTORY(g2o)
+
+
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/libg2o)
-- 
1.8.1.2

