From 73aff98c9419e81b03361b5c0a8c7f4a387c9ed2 Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Thu, 8 Nov 2012 15:50:10 -0800
Subject: [PATCH 11/11] add the missing libraries

---
 CMakeLists.txt |   14 +++++++++-----
 1 file changed, 9 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4405d44..489d225 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -243,13 +243,17 @@ ADD_SUBDIRECTORY(g2o)
 
 project(libg2o)
 find_package(catkin)
+
+set(FILES "")
+foreach(file cli core csparse_extension ext_freeglut_minimal incremental interactive interface opengl_helper parser
+             simulator solver_cholmod solver_csparse solver_dense solver_pcg solver_slam2d_linear solver_structure_only stuff types_data types_icp types_sba
+             types_sclam2d types_sim3 types_slam2d types_slam3d)
+list(APPEND FILES ${LIB_PREFIX}${file})
+endforeach()
 if (${catkin_VERSION} VERSION_GREATER "0.5.28")
-catkin_package(INCLUDE_DIRS include
-               LIBRARIES ${LIB_PREFIX}core ${LIB_PREFIX}types_slam3d ${LIB_PREFIX}types_icp ${LIB_PREFIX}stuff ${LIB_PREFIX}csparse_extension
+catkin_package(LIBRARIES ${FILES}
 )
 else()
 catkin_stack()
-catkin_project(libg2o INCLUDE_DIRS include
-                      LIBRARIES ${LIB_PREFIX}core ${LIB_PREFIX}types_slam3d ${LIB_PREFIX}types_icp ${LIB_PREFIX}stuff ${LIB_PREFIX}csparse_extension
-)
+catkin_project(libg2o LIBRARIES ${FILES})
 endif()
-- 
1.7.10.4

